### PR TITLE
DOC-667 Add counter function

### PR DESCRIPTION
**Pull request by asimms41**
## Description

Resolves: [DOC-667](https://redpandadata.atlassian.net/browse/DOC-667)
Review deadline: 30th October

Fixes bug caused by Bloblang `count()` function. In all cases, replaced by `counter()` function.

## Page previews

**Outputs**

- [aws_s3](https://deploy-preview-80--redpanda-connect.netlify.app/redpanda-connect/components/outputs/aws_s3/)
- [azure_blob_storage](https://deploy-preview-80--redpanda-connect.netlify.app/redpanda-connect/components/outputs/azure_blob_storage/)
- [elasticsearch](https://deploy-preview-80--redpanda-connect.netlify.app/redpanda-connect/components/outputs/elasticsearch/)
- [gcp_cloud_storage](https://deploy-preview-80--redpanda-connect.netlify.app/redpanda-connect/components/outputs/gcp_cloud_storage/)
- [hdfs](https://deploy-preview-80--redpanda-connect.netlify.app/redpanda-connect/components/outputs/hdfs/)
- [redis_hash](https://deploy-preview-80--redpanda-connect.netlify.app/redpanda-connect/components/outputs/redis_has/)
- [redis_list](https://deploy-preview-80--redpanda-connect.netlify.app/redpanda-connect/components/outputs/redis_list/)

## Checks

- [ ] New feature
- [x] Content gap
- [ ] Support Follow-up
- [ ] Small fix (typos, links, copyedits, etc)

[DOC-667]: https://redpandadata.atlassian.net/browse/DOC-667?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by mihaitodor — APPROVED** [review]
LGTM :shipit:

**Review by Jeffail — APPROVED** [review]
LGTM

File: modules/components/pages/outputs/aws_s3.adoc [hunk 1]
```diff
@@ -116,15 +116,15 @@ By default Redpanda Connect will use a shared credentials file when connecting t
 
 == Batching
 
-It's common to want to upload messages to S3 as batched archives, the easiest way to do this is to batch your messages at the output level and join the batch of messages with an xref:components:processors/archive.adoc[`archive`] and/or xref:components:processors/compress.adoc[`compress`] processor.
+It's common to want to upload messages to S3 as batched archives. The easiest way to do this is to batch your messages at the output level and join the batch of messages with an xref:components:processors/archive.adoc[`archive`] and/or xref:components:processors/compress.adoc[`compress`] processor.
```
Comment (kbatuigas) [inline modules/components/pages/outputs/aws_s3.adoc:119]:
```suggestion
It's common to want to upload messages to S3 as batched archives. The easiest way to do this is to batch your messages at the output level and join the batch of messages with an xref:components:processors/archive.adoc[`archive`] or xref:components:processors/compress.adoc[`compress`] processor.
```
Per style guide. Happy to go with `and` as well.

**Review by kbatuigas — APPROVED** [review]
One suggested edit, but no blockers otherwise.

File: modules/components/pages/outputs/aws_s3.adoc (hunk 1 above)
Comment (asimms41) [reply modules/components/pages/outputs/aws_s3.adoc]: Updated.

**Pull request merged**